py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
Travis say this is okay for python 2.7? I think flake8 will complain that I imported this, but not using it. Will put it in if you confirm go ahead one more time.